keyboards/splitkb/halcyon/kyria/rev4/keyboard.json

```diff
@@ -0,0 +1,170 @@
+            {"matrix": [2, 6], "x": 0, "y": 41, "flags": 4},
+            {"matrix": [1, 1], "x": 72, "y": 23, "flags": 4},
+            {"matrix": [1, 2], "x": 58, "y": 19, "flags": 4},
+            {"matrix": [1, 3], "x": 43, "y": 15, "flags": 4},
+            {"matrix": [1, 4], "x": 29, "y": 19, "flags": 4},
+            {"matrix": [1, 5], "x": 14, "y": 26, "flags": 4},
+            {"matrix": [1, 6], "x": 0, "y": 26, "flags": 4},
+            {"matrix": [0, 1], "x": 72, "y": 8, "flags": 4},
+            {"matrix": [0, 2], "x": 58, "y": 4, "flags": 4},
+            {"matrix": [0, 3], "x": 43, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 29, "y": 4, "flags": 4},
+            {"matrix": [0, 5], "x": 14, "y": 11, "flags": 4},
+            {"matrix": [0, 6], "x": 0, "y": 11, "flags": 4},
+            {"x": 149, "y": 2, "flags": 2},
+            {"x": 174, "y": 1, "flags": 2},
+            {"x": 210, "y": 4, "flags": 2},
+            {"x": 199, "y": 45, "flags": 2},
+            {"x": 166, "y": 49, "flags": 2},
+            {"x": 130, "y": 53, "flags": 2},
+            {"matrix": [7, 0], "x": 130, "y": 64, "flags": 4},
+            {"matrix": [7, 5], "x": 145, "y": 60, "flags": 4},
+            {"matrix": [7, 1], "x": 159, "y": 53, "flags": 4},
+            {"matrix": [7, 2], "x": 173, "y": 49, "flags": 4},
+            {"matrix": [7, 4], "x": 188, "y": 49, "flags": 4},
+            {"matrix": [6, 0], "x": 123, "y": 49, "flags": 4},
+            {"matrix": [7, 3], "x": 137, "y": 45, "flags": 4},
+            {"matrix": [6, 1], "x": 152, "y": 38, "flags": 4},
+            {"matrix": [6, 2], "x": 166, "y": 34, "flags": 4},
+            {"matrix": [6, 3], "x": 181, "y": 30, "flags": 4},
+            {"matrix": [6, 4], "x": 195, "y": 34, "flags": 4},
+            {"matrix": [6, 5], "x": 210, "y": 41, "flags": 4},
+            {"matrix": [6, 6], "x": 224, "y": 41, "flags": 4},
+            {"matrix": [5, 1], "x": 152, "y": 23, "flags": 4},
+            {"matrix": [5, 2], "x": 166, "y": 19, "flags": 4},
+            {"matrix": [5, 3], "x": 181, "y": 15, "flags": 4},
+            {"matrix": [5, 4], "x": 195, "y": 19, "flags": 4},
+            {"matrix": [5, 5], "x": 210, "y": 26, "flags": 4},
+            {"matrix": [5, 6], "x": 224, "y": 26, "flags": 4},
+            {"matrix": [4, 1], "x": 152, "y": 8, "flags": 4},
+            {"matrix": [4, 2], "x": 166, "y": 4, "flags": 4},
+            {"matrix": [4, 3], "x": 181, "y": 0, "flags": 4},
+            {"matrix": [4, 4], "x": 195, "y": 4, "flags": 4},
+            {"matrix": [4, 5], "x": 210, "y": 11, "flags": 4},
+            {"matrix": [4, 6], "x": 224, "y": 11, "flags": 4}
+        ],
+        "max_brightness": 128,
+        "sleep": true,
+        "split_count": [31, 31]
+    },
+    "split": {
+        "enabled": true,
+        "handedness": {
+            "pin": "GP24"
+        },
+        "matrix_pins": {
+            "right": {
```

Comment:
It looks like the only difference with the normal pin config  is that the column order is reversed.  This could be handled by changing the column order in the layout itself, which would reduce complicity, a bit.

Reply:
I tried this but it gave some weird behavior with columns being shifted. I also feel like this makes it a bit more complex as it will then be different from how our schematics are wired up from a troubleshooting standpoint. Keeping it like this feels like the better solution to me.